Add \mathit where needed

andreas-junghanns opened the pull request:
I added \mathit where needed, I think.

andreas-junghanns commented:
This addresses #915. I fail to link this issue in the side bar... no idea what is wrong.

clagms commented on docs/2_1_common_api.adoc:
```diff
@@ -500,9 +500,9 @@ All elements of the array, including structural zeros, are serialized.
-A = \left( \begin{array}{cc} a_{11}&a_{12}\\
-                             a_{21}&a_{22}\\
-                             a_{31}&a_{32}\\
+A = \left( \begin{array}{cc} a_{\mathit{11}}&a_{\mathit{12}}\\
```
I think there's no need to use \mathit on numbers. It's only on text, as that's where latex does funny stuff.
This comment applies to everywhere else where changes were made.

andreas-junghanns replied:
fixed

clagms commented on docs/2_1_common_api.adoc:
```diff
@@ -587,19 +587,19 @@ A clocked partition is mathematically defined as:
 [latexmath]
 ++++
-\text{with } j=0,1,2,...;x_{-1}=x_{start}
+\text{with } j=0,1,2,...;x_{\mathit{-1}}=x_{\mathit{start}}
```
Here, only \mathit{start} is correct.

andreas-junghanns replied:
fixed

clagms commented on docs/2_1_common_api.adoc:
```diff
@@ -960,12 +960,12 @@ In every state, the general form of the FMU equations are:
 where
 
-* latexmath:[\mathbf{v}_{unknonwn}] is the vector of unknown floating point variables computed in the actual state:
+* latexmath:[\mathbf{v}_{\mathit{unknonwn}}] is the vector of unknown floating point variables computed in the actual state:
```
Typo here: is it on purpose that we're keeping that typo?

andreas-junghanns replied:
fixed

clagms commented on docs/2_2_common_schema.adoc:
```diff
@@ -1912,22 +1912,22 @@ In other words, the respective partial derivatives cannot be computed.
+Otherwise, the corresponding known latexmath:[\color{blue}{v_{\mathit{known,i}}}] enters the equation as:
 
-`= dependent`: no particular structure, latexmath:[\color{blue}{{h(..,\ v}_{known,i}}],..)
+`= dependent`: no particular structure, latexmath:[\color{blue}{{h(..,\ v}_{\mathit{known,i}}}],..)
```
Maybe this was find and replace magic, but 
`_{\mathit{known,i}}`
can be made more correct if transformed into:
`_{\mathit{known}, i}`
as `i` is a different element.

andreas-junghanns replied:
fixed

clagms commented on docs/2_4_common_co-simulation.adoc:
`\mathit{i+1}` is incorrect. Should be only `i+1`. 
Mathit should be used one for words that are meant to denote the same mathematical concept.

andreas-junghanns replied:
fixed

andreas-junghanns commented:
@clagms : why do you list these issues here and not fix it right in the PR?
Listing here makes this extremely inefficient and your comments are likely all valid, so they should not be treated as proposals but as improvements.

andreas-junghanns commented:
@clagms : I addressed all your comments. Can you please check again (and approve)?
I would like to merge soon as this touches many lines and merge conflicts with future PRs are likely.

clagms commented:
This is not fixed yet @andreas-junghanns , but I'll do it!

clagms commented:
In the equations near ``An example of use for an array of stream variables is a gas mixture ``, the `\textit` command should be replaced by `\mathit`. However, because I don't know what the `[]` subscript means, I'll leave those equations as they are.
Here's a picture:
![image](https://user-images.githubusercontent.com/871103/82202469-3ec05d80-9902-11ea-86a1-ff8add2e5c2b.png)

clagms commented:
I fixed the mathit's that were introduced incorrectly as part of the regex magic.
I did not check whether there are math elements that should have a mathit in them (I assume @andreas-junghanns did not miss any of these).
